tests/pytest.ini

```diff
@@ -1,5 +1,8 @@
 [pytest]
 markers =
+    unit_test: marks tests as unit-level tests (select with '-m "unit_test"')
+    integration_test: marks tests as integration-level tests (select with '-m "integration_test"')
+    system_test: marks tests as system-level tests (select with '-m "system_test"')
```

Comment:
There are some existing markers that could be relevant, e.g., `core`, `slow`. Is our plan going forward to keep everything, or eventually phase out the `core` and `slow` for `unit_test`, etc.?

Reply:
I think we can keep the existing --- multiple markers are valid, and can serve different purposes. 
For example, a unit test may be slow, and a system-level test may be fast, so we should probably avoid conflating them.

Reply:
Generally agree, but it is another axis of options that devs have to remember when marking tests.

Reply:
This will require being rolled out across the org with best practice guidelines. I don't see how we can achieve that in the testing frameworks without us defining the tests as matching the associated buckets we want to achieve test coverage in.